controls/gridview/getting-started.md

```diff
@@ -13,78 +13,101 @@ previous_url: gridview-getting-started
 
 | RELATED VIDEOS |  
 | ------ |
-|[RadGridView for WinForms Webinar](http://tv.telerik.com/watch/winforms/webinar/radgridview-for-winforms-webinar)<br>Lino Tadros from Falalel provides an in-depth overview of Telerik's RadGridView for WinForms (Runtime: 01:45:11)|
+|[RadGridView for WinForms Webinar](https://www.telerik.com/videos/winforms/radgridview-for-winforms-webinar)<br>Lino Tadros from Falalel provides an in-depth overview of Telerik's RadGridView for WinForms (Runtime: 01:45:11)|
 
-The example below demonstrates creating a hierarchical grid presenting two tables *Product Category* and *Product Information*.
+The following tutorial demonstrates how to create a hierarchical grid that presents two tables: *Product Category* and *Product Information*.
 
->caption Figure 1: The final result from the tutorial.
+>caption Figure 1: The final result
 
 ![grid-getting-started 001](images/grid-getting-started001.png)
 
-## Setting up the Form 
+## Setting up the Form
+
+1. Create a new __Windows Forms Application__ project.
 
-1. Create a new __Windows Application__ project.
-            
 1. Drop a __RadGridView__ control on the form.
 
-1. Set the RadGridView.__Dock__ property to *Fill* in the Properties window of Visual Studio.
+1. In the __Properties__ window for __RadGridView__, set the `RadGridView.Dock` property to *Fill*.
 
 ## Connecting to Data
 
-1. Open the __DataSource__ property drop down in the *Properties* window of Visual Studio and select __Add Project Data Source__. This step will open the __Data Source Configuration Wizard__.
+1. In the __Properties__ window for __RadGridView__, locate the `DataSource` property drop-down.
+
+1. Select __Add Project Data Source__. This step will open the __Data Source Configuration Wizard__.
 
-1. In the **Data Source Configuration Wizard**, Choose a Data Source Type page, select the __Database__ icon, and click __Next__.
+1. On the __Choose a Data Source Type__ page, select __Database__ and click __Next__.
 ![grid-getting-started 002](images/grid-getting-started002.png)
 
-1. In the Choose Your Data Connection page of the wizard, click on the __New Connection...__ button. This page displays the `Add Connection` dialog.
```

Comment:
When I go through the tutorial, I see an additional __Choose a Database Model__ page. I suggest adding it.